api/envoy/config/filter/network/tcp_proxy/v2/tcp_proxy.proto

```diff
@@ -148,4 +148,21 @@ message TcpProxy {
   // The maximum number of unsuccessful connection attempts that will be made before
   // giving up. If the parameter is not specified, 1 connection attempt will be made.
   google.protobuf.UInt32Value max_connect_attempts = 7 [(validate.rules).uint32 = {gte: 1}];
+
+  // Specifies the TCP proxy's hashing policy
```

Comment:
I am actually going to need this for UDP proxy as well. Can you do me a favor and move this into a common file that I can include? Potentially somewhere in types? Maybe a new hash file there?

Reply:
Done.